source/CMakeLists.txt

```diff
@@ -17,10 +17,10 @@ target_sources(unifex
     inplace_stop_token.cpp
     manual_event_loop.cpp
     static_thread_pool.cpp
+    task.cpp
     thread_unsafe_event_loop.cpp
     timed_single_thread_context.cpp
-    trampoline_scheduler.cpp
-    async_manual_reset_event.cpp)
```

Comment:
Is this a bad merge?

Reply:
````
 target_sources(unifex
   PRIVATE
     async_auto_reset_event.cpp
     async_manual_reset_event.cpp
     async_mutex.cpp
     exception.cpp
     inplace_stop_token.cpp
     manual_event_loop.cpp
     static_thread_pool.cpp
     task.cpp
     thread_unsafe_event_loop.cpp
     timed_single_thread_context.cpp
     trampoline_scheduler.cpp)
```

resulting block, both `.cpp` files are still listed